docs/community-ee/community-ee-release-process.md

```diff
@@ -11,68 +11,65 @@ The EE versioning convention is core tag plus patch number, for example:
     - EE with core `2.16.2` comes out -> `community-ee:2.16.2-1`
     - EE with core `2.16.3` comes out -> `community-ee:2.16.3-1`
 
-## Dependencies
-
-- Podman/Docker
-
 ## Credentials
 
-- Access to the https://github.com/ansible-community/images repository
-- Access to ghcr.io
+- Access to https://github.com/ansible-community/images repo.
```

Comment:
When you say "access" to a repository, does that mean having maintainer permissions? Or does it just mean have a fork / clone?